packages/fxa-content-server/app/images/graphic-logo-relay.svg

```diff
@@ -0,0 +1,21 @@
+<svg width="434" height="80" viewBox="0 0 195 43" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Not sure where you got graphic, but I usually like to run it through https://jakearchibald.github.io/svgomg/ to get an optimized image.

Reply:
The graphic was supplied in jira.

Reply:
i got the graphic from UX and added it to the jira. just fyi

Reply:
Alright, the graphic has been optimized.